tests/Functional/RolloutPercentageTestCase.php

```diff
@@ -23,7 +23,21 @@
     $this->assertGreaterThanOrEqual(240, $rollout['variant']); // 48
 });
 
+test('small list should have a correct percentage rollout', function () {
+    $variantRetriever = generateVariantRetriever();
+
+    $results = [];
+    foreach (range(1, 100) as $value) {
+        $results[] = $variantRetriever->getVariantForExperiment(new Experiment('my-ab-test'), (string)$value);
+    }
 
+    $this->assertCount(100, $results);
```

Comment:
Why this assertion? It's obvious that there will be 100 results as you add 100 values just above